modules/user_accounts/templates/form_edit_user.tpl

```diff
@@ -297,89 +297,117 @@ $(document).ready(function() {
-    		{$form.PermID_Group.label}
-    	</label>
-    	<div class="col-sm-10 col-xs-12">
-    		<div>
-    		{$form.PermID_Group.html}
-    		</div>
-    	</div>
-    </div>
-    <div class="row form-group form-inline">
-        <label class="col-sm-2">
-          {$form.Supervisors_Group.label}
-        </label>
-        <div class="col-sm-10 col-xs-12">
-          <div>
-            {$form.Supervisors_Group.html}
+        <div class="row form-group form-inline">
+           <label class="col-sm-2">
+              {$form.Active.label}
+          </label>
+          <div class="col-sm-10">
+              {$form.Active.html}
           </div>
-        </div>
-     </div>
+      </div>
+      <div class="row form-group form-inline">
+       <label class="col-sm-2">
+          {$form.Pending_approval.label}
+      </label>
+      <div class="col-sm-10">
+          {$form.Pending_approval.html}
+      </div>
+  </div>
+  <div class="row form-group form-inline">
+   <label class="col-sm-2">
+      {$form.PermID_Group.label}
+  </label>
+  <div class="col-sm-10 col-xs-12">
+      <div>
+          {$form.PermID_Group.html}
+      </div>
+  </div>
+</div>
+<div class="row form-group form-inline">
+    <label class="col-sm-2">
+      {$form.Supervisors_Group.label}
+  </label>
+  <div class="col-sm-10 col-xs-12">
+      <div>
+        {$form.Supervisors_Group.html}
+    </div>
+</div>
+</div>
+<div class="row form-group form-inline">
+   <div class="col-sm-2">
+      <input class="btn btn-sm btn-primary col-xs-12" name="fire_away" value="Save" type="submit" />
+  </div>
+  <div class="col-sm-2">
+    <input class="btn btn-sm btn-primary col-xs-12" value="Reset" type="reset"/>
+</div>
+<div class="col-sm-2">
+  <input class="btn btn-sm btn-primary col-xs-12" onclick="location.href='{$baseurl}/user_accounts/'" value="Back" type="button" />
+</div>
+{if $can_reject}
 
-    <div class="row form-group form-inline">
-    	<div class="col-sm-2">
-    		<input class="btn btn-sm btn-primary col-xs-12" name="fire_away" value="Save" type="submit" />
-    	</div>
-    	<div class="col-sm-2">
-    		<input class="btn btn-sm btn-primary col-xs-12" value="Reset" type="reset" />
-    	</div>
-    	<div class="col-sm-2">
-    		<input class="btn btn-sm btn-primary col-xs-12" onclick="location.href='{$baseurl}/user_accounts/'" value="Back" type="button" />
-    	</div>
+<div class="col-sm-2">
+    <input type=hidden id ="UserID" value="{$form.UserID.html}">
+    <input type=hidden id = "baseurl" value="{$baseurl}">
+    <input class="btn btn-sm btn-primary col-xs-12" value="Reject User" type="button" id="btn_reject"/>
+    {literal}
+    <script type="text/javascript">
+        $(document).ready(
+            function(){
+                $("#btn_reject").click(
+                    function(){
+                        var userID = document.getElementById("UserID").value;
+                        var baseurl = document.getElementById("baseurl").value;
```

Comment:
@driusan  I'm able to get the url from ``` loris.BaseURL ``` but I'm not able to access the form object, which holds the UserID field. Should I keep that one as a hidden field?